subsys/usb/usb_transfer.c

```diff
@@ -0,0 +1,327 @@
+/*
+ * Copyright (c) 2018 Linaro
+ * Copyright (c) 2019 PHYTEC Messtechnik GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <usb/usb_device.h>
+#include <logging/log.h>
+
+#include "usb_transfer.h"
+
+LOG_MODULE_REGISTER(usb_transfer, CONFIG_USB_DEVICE_LOG_LEVEL);
+
+#define MAX_NUM_TRANSFERS           4 /** Max number of parallel transfers */
+
+struct usb_transfer_sync_priv {
```

Comment:
I would place it in header file.

Reply:
No need to move code to header unless it is used somewhere else.

Reply:
That's right